Resources/doc/index.rst

```diff
@@ -16,6 +16,11 @@ following command to download the latest stable version of this bundle:
 
     composer require --dev doctrine/doctrine-fixtures-bundle
 
+.. tip::
+
+    If you are using flex you just need to do:
+    ``composer req orm-fixtures``
```

Comment:
Should this also include `--dev` to be consistent with the `require` command above?

Reply:
You are right. Thanks!